plugins/chat/db/post_migrate/20240516145911_update_user_options_for_thread_title_prompts.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class UpdateUserOptionsForThreadTitlePrompts < ActiveRecord::Migration[7.0]
+  def up
+    change_column_default :user_options, :show_thread_title_prompts, true
+
+    batch_size = 100_000
+    min_id = DB.query_single("SELECT MIN(user_id) FROM user_options").first.to_i
+    max_id = DB.query_single("SELECT MAX(user_id) FROM user_options").first.to_i
+    while max_id > min_id
```

Comment:
```suggestion
    while max_id >= min_id
```

To avoid the (extremely unlikely) case where `min_id` is 1 and `max_id` is 100,001 which we would result in `user_id` 1 missing this update.